QNX change from qnx-ports/build-files, copied unchanged:
  ports/ros2/ros2-jazzy/qnx_patches/rcl_interfaces.patch
  at revision 6f1507550d656edafbf8bcc826a196da1b00c459.

Applied by ports/ros2/recipe.sh, in the repository repo-patches/MAP names.
The reasons for the individual changes are in that file's history in
qnx-ports/build-files.

Upstream-Status: QNX-specific

diff --git a/builtin_interfaces/package.xml b/builtin_interfaces/package.xml
index b09b90a..f95cf57 100644
--- a/builtin_interfaces/package.xml
+++ b/builtin_interfaces/package.xml
@@ -17,6 +17,8 @@
 
   <buildtool_depend>rosidl_core_generators</buildtool_depend>
 
+  <depend>numpy_vendor</depend>
+
   <exec_depend>rosidl_core_runtime</exec_depend>
 
   <test_depend>ament_lint_common</test_depend>
diff --git a/lifecycle_msgs/package.xml b/lifecycle_msgs/package.xml
index 2dc4ce5..e185a67 100644
--- a/lifecycle_msgs/package.xml
+++ b/lifecycle_msgs/package.xml
@@ -16,6 +16,8 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
+  <depend>numpy_vendor</depend>
+
   <exec_depend>rosidl_default_runtime</exec_depend>
 
   <test_depend>ament_lint_auto</test_depend>
